extensions/amazon-aws/src/console.tsx

```diff
@@ -23,15 +23,7 @@ export default function Console() {
           keywords={service.match.split(" ")}
           actions={
             <ActionPanel>
-              <AwsAction.Console
-                url={
-                  typeof process.env.AWS_SSO_ACCOUNT_ID !== "undefined" &&
-                  typeof process.env.AWS_SSO_ROLE_NAME !== "undefined" &&
-                  typeof process.env.AWS_SSO_START_URL !== "undefined"
-                    ? `${normalizeUrl(process.env.AWS_SSO_START_URL)}console?account_id=${encodeURI(process.env.AWS_SSO_ACCOUNT_ID)}&role_name=${encodeURI(process.env.AWS_SSO_ROLE_NAME)}&destination=${encodeURI(AWS_URL_BASE + service.arg)}`
-                    : `${AWS_URL_BASE}${service.arg}`
-                }
-              />
+              <AwsAction.Console url={`${AWS_URL_BASE}${service.arg}`} />
```

Comment:
suggestion (non-blocking): Console command is a perfect candidate for [frecency sorting](https://github.com/raycast/extensions/issues/13025). You might need to rebase based on these changes.